test/cmd/templates.sh

```diff
@@ -76,6 +79,9 @@ echo "template data precision: ok"
 
 os::cmd::expect_success 'oc create -f examples/sample-app/application-template-dockerbuild.json -n openshift'
 os::cmd::expect_success 'oc policy add-role-to-user admin test-user'
+new="$(mktemp -d)/tempconfig"
+os::cmd::expect_success "oc config view --raw > $new"
+export KUBECONFIG=$new
```

Comment:
Because the point of this change is to not require you to run hack/test-cmd.sh to test this.  You just run test/cmd/templates.